hawk/app/assets/javascripts/application.js

```diff
@@ -54,14 +52,24 @@ $(function() {
     );
   };
 
+  var createMutationObserver = function() {
+    var target = document.querySelector('#middle');
+    var observer = new MutationObserver(function(mutations) {
+      observer.disconnect();
```

Comment:
You are accessing `observer` which gets defined outside with the? Looks wired...